server/openapi_server/__main__.py

```diff
@@ -2,6 +2,7 @@
 
 import connexion
 from openapi_server import encoder
+from openapi_server.util.configuration import CD2HConfig as config
```

Comment:
Rename`CD2HConfig` to something more generic, e.g. `config`. We will not have multiple configuration so there is no need to be too specific. Making the name more generic will also make it more friendly to developer who will reuse it.